packages/siopv2-oid4vp-rp-rest-client/src/agent/SiopV2OID4VpRpRestClient.ts

```diff
@@ -0,0 +1,96 @@
+import { fetch } from 'cross-fetch'
+import {
+  ISiopV2OID4VpRpRestClient,
+  IGenerateAuthRequestArgs,
+  IGenerateAuthRequestURIResponse,
+  IGetAuthStatusArgs,
+  IRequiredContext,
+  IRemoveAuthRequestSessionArgs,
+} from '../index'
+import Debug from 'debug'
+import { IAgentPlugin } from '@veramo/core'
+
+const debug = Debug('ssi-sdk-siopv2-oid4vp-rp-rest-client:SiopV2OID4VpRpRestClient')
+// todo merge this with Niels's branch feature/siop-verifier and use those classes/definitions
+export class SiopV2OID4VpRpRestClient implements IAgentPlugin {
+  readonly methods: ISiopV2OID4VpRpRestClient = {
+    removeAuthRequestSession: this.removeAuthRequestSession.bind(this),
```

Comment:
please rename these methods, as they will end up as global methods in a configured agent. See the siopv2-oid4vp-auth package for naming. Should become something like 

siopClientRemoveAuthRequestSession or something

Reply:
changed to `siopClientRemoveAuthRequestSession` also changed other two method names to a more specific name